tests/dummy/app/templates/application.hbs

```diff
@@ -6,4 +6,9 @@
   </div>
 {{/code-block}}
 
+<h3>With `line-numbers` plugin:</h3>
+{{#code-block class="line-numbers"}}&lt;div&gt;
```

Comment:
I feel like instead of just using `class`, this should be `plugins` or something. If only to be more explicit.
I can address that in a separate PR, but @justin-lau what do you think about that?